single_tech_samples/purview/sample2_data_plane_permissions_release_mng/README.md

```diff
@@ -0,0 +1,65 @@
+# Github Action to Set Purview Permissions
+
+A github action to manage permissions with purview.
+
+## Problem Statement
+
+Managing purview permissions cannot be easily done programmatically because permissions are managed in the data plane and not in IAM. This makes it difficult to assign permissions using IaCs such as terraform or bicep. Also, the only permission currently available to assign via `az cli` is the root collection admin.
+
+This action allows you to assign any purview permission(s) to a user or security group.
+
+## Sample Usage
+
+> *Note: It is preferable to clone or fork this action when testing to avoid populating the actions log for this repo.*
+
+To use the action in a workflow with a service principal for example, you can [create a service principal](https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-portal%2Clinux#create-a-service-principal), connect to github and ensure right permissions to successfully run the workflow.
+
+### Ensuring Right Permissions
+
+The identity running the workflow needs to have the following permissions:
+
+- A minimum of the `Contributor` and `User Access Administrator` roles on the subscription containing the purview instance. [See how to assign roles to an identity](https://learn.microsoft.com/en-us/cli/azure/role/assignment?view=azure-cli-latest)
+
+- The identity must also have the `root collection admin` access on the purview instance. [See how to add root collection admin to purview](https://learn.microsoft.com/en-us/cli/azure/purview/account?view=azure-cli-latest#az-purview-account-add-root-collection-admin)
+
+### Sample Workflow
+
+The action can be used in your github workflows like below:
+
+```yaml
+name: Test pipeline
+
+on: [push]
+
+jobs:   
+  workflow-test:
+    name: 'Set purview permissions with workflow'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: 'Az login'
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Set purview permissions
+        uses: abeebu/purview-custom-permissions@v1   # or uses: ./ if you are using the source code in your repository
```

Comment:
replace uses: abeebu/purview-custom-permissions@v1 by uses:<owner>/purview-custom-permissions@v1

Reply:
@ydaponte But the `uses:/purview-custom-permissions@v1` does not exist on the github marketplace. Would it still be okay to replace it with that. To be able to use an action, we either need to have an absolute url (meaning that it has been deployed on the marketplace) or we can use a relative url like `./`